src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -201,6 +209,41 @@ export default class CoveredEResourcesList extends React.Component {
     );
   }
 
+  renderPtiUrl = (eresource) => {
+    const url = eresource?._object?.pti?.url;
+    return url ? (
+      <Tooltip
+        key={eresource.id}
```

Comment:
Why are we setting a `key` here?

Reply:
🤦  ...Its actually a List of rows with Tooltips vs List of toolTips.